osaca/parser/label.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+from osaca.parser.operand import Operand
+
+
+class LabelOperand(Operand):
+    def __init__(self, name=None, comment_id=None):
+        super().__init__(name)
+        self._comment_id = comment_id
```

Comment:
I think the comment should be always the attribute of an `InstructionForm` object and not of any of the operands. Removing it here (and in any other operand in case I missed it) would make sense I believe.

Reply:
https://github.com/RRZE-HPC/OSACA/commit/7ad3438af543c9d6991cac6fed9450ab6adb1500